lib/network/modules/components/Edge.js

```diff
@@ -144,6 +144,23 @@ class Edge {
     // only deep extend the items in the field array. These do not have shorthand.
     util.selectiveDeepExtend(fields, parentOptions, newOptions, allowDeletion);
 
+    // Only use endPointOffset values (from and to) if it's valid values
+    if (newOptions.endPointOffset !== undefined) {
+      if (ComponentUtil.isValidInteger(newOptions.endPointOffset.from)) {
```

Comment:
This is just a question at this point, not a request for change: Why are only integers valid? Would something terrible happen if it was for example 1.2?

Reply:
I have just test with integers but it should not matters. Do you think there is a need for floating numbers? Maybe if the place of the endpoints have to be very exactly..?

Reply:
I change so also floats is valid.